src/core/analyzer/quality_reporter.py

```
@@ -291,10 +238,7 @@ def _show_grouped_primitive_issues(
         # 除外グループ（汎用変数名）
         if "excluded" in grouped:
             excluded_issues = grouped["excluded"]
-            self.console.print(
-                f"[bold {color}]汎用変数名（型定義不要） "
-                f"({len(excluded_issues)}件)[/bold {color}]"
-            )
+            self.console.print(f"[bold {color}]汎用変数名（型定義不要） ({len(excluded_issues)}件)[/bold {color}]")
```

Ruff (0.14.0)

String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)